packages/backend/src/services/OrganizationService/OrganizationService.ts

```diff
@@ -926,7 +926,7 @@ export class OrganizationService extends BaseService {
                         organizationUuid,
                         expiresAt: setup.serviceAccount.expirationTime,
                         description: 'Initial setup service account',
-                        scopes: [ServiceAccountScope.ORG_ADMIN],
+                        scopes: [ProjectScope.MANAGE],
```

Comment:
how came this got downgraded from org admin to project manage? does it mean it can't manage the organization?

Reply:
Yes, we're optimizing towards more granular access. Javi and I both agreed that smaller granular access control would be better here rather than broad access before we know more about how Kraken will use scopes.